Developer

```diff
@@ -0,0 +1,90 @@
+#######################################
+# Developer file for host development machine
+#######################################
+
+# GNU core utilities (those that come with OS X are outdated)
+# Don’t forget to add `$(brew --prefix coreutils)/libexec/gnubin` to `$PATH`.
+brew 'coreutils'
+# GNU `find`, `locate`, `updatedb`, and `xargs`, `g`-prefixed
+brew 'findutils'
+# GNU `sed`, overwriting the built-in `sed`
+brew 'gnu-sed'
+# Latest OpenSSL
+brew 'openssl'
+# Latest Bash
+# Note: don’t forget to add `/usr/local/bin/bash` to `/etc/shells` before running `chsh`.
+brew 'bash'
+# Bash completion
+brew 'bash-completion'
+
+# Useful completions
+brew 'brew-cask-completion'
+brew 'bundler-completion'
+brew 'gem-completion'
+brew 'open-completion'
+brew 'vagrant-completion'
+
+# Latest versions of essential tools
+brew 'wget'
+brew 'curl'
+brew 'git'
+
+# PHP and tools
+brew 'php71'
+
+# Languages
+brew 'node' # This installs `npm` too using the recommended installation method
+brew 'python3'
+brew 'rbenv'
+
+# JavaScript package manager
+brew 'yarn'
+
+# Image-related tools
+brew 'jpegoptim'
+brew 'optipng'
+brew 'imagemagick'
+
+# For GUI and other apps having an installer
+tap 'caskroom/cask'
+tap 'caskroom/versions'
+
+# Fonts
+tap 'caskroom/fonts'
+cask 'font-source-code-pro'
+
+# Browsers
+cask 'firefox'
+cask 'google-chrome'
+
+# Google Drive
+cask 'google-drive-file-stream'
+
+# Communication
+cask 'slack'
+
+# UI & vectors
+cask 'sketch'
+
+# Text-editors
+brew 'vim'
+tap 'neovim/neovim'
+brew 'neovim'
```

Comment:
```suggestion
```
Let's remove neovim. If someone really needs it, they can install it, but it creates more problems than it solves.

Reply:
@hkdobrev I didn't know that. Can you please clarify what problems ?

Reply:
@BaselAl Mostly that neovim is not used and is not as easy to configure. 

Even if vim is not widely used, this formula replaces the normal vi in MacOS by default with a newer version of vim. However, since we don't have active neovim users, there's no need to overcomplicate things. Power vim configuration users, can install neovim additionally.

Reply:
This suggestion still stands.

Reply:
Hi @hkdobrev   
I've done it.

Reply:
@BaselAl You can then resolve the conversation :)